.flake8

```
@@ -6,11 +6,11 @@ ignore=
     # Missing Docstrings
     D100,D104,D107,
     # Docstring Whitespace
-    D202,D203,D204,D212,D214,D215,
+    D203,D212,D214,D215,
     # Docstring Quotes
     D301,D302,
     # Docstring Content
-    D400,D401,D402,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
+    D400, D401,D402,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
```

This will endlessly bother me otherwise.
```suggestion
    D400,D401,D402,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
```